woocommerce/woocommerce-gutenberg-products-block
# Mini Cart Block: add link to edit template part from sidebar

This PR renders a link on the right sidebar to allow the user to land on the page to edit the `Mini Cart` template part.




Fixes #5186 





### Screenshots

![image](https://user-images.githubusercontent.com/4463174/142662783-89d37afc-8941-444b-b60f-215d5e5aa5ae.png)


### Testing

### Manual Testing

How to test the changes in this Pull Request:

Check out this branch 

1. Make sure you have Woo Blocks plugins installed and a theme that supports FSE.
2. Create a page.
3. Add the `Mini Cart` block and click on it.
4. On the right sidebar, in the block section, check if there is a `Template Editor` menu.
5. Check that there is a link and the link structure is `/wp-admin/themes.php?page=gutenberg-edit-site&postId=$SLUG_THEME%2F%2Fmini-cart&postType=wp_template_part`.
6. Change the theme and use a theme that doesn't support FSE.
7. Create a page.
8. Add the `Mini Cart` block and click on it.
9. On the right sidebar, in the block section, check if there isn't the `Template Editor` menu.


### Changelog
> Add link to edit template part from Mini Cart block sidebar when it is utilized a theme that supports FSE.
